### PR TITLE
Changelog 1.11.0

### PR DESCRIPTION
**Pull request by jolheiser**
As title :tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,77 @@ This changelog goes through all the changes that have been made in each release
...
+  * Hide some user information via API if user have no enough permission (#8655)
+  * Move startpage/homepage translation to crowdin (#8596)
 * SECURITY
   * Never allow an empty password to validate (#9682) (#9683)
   * Prevent redirect to Host (#9678) (#9679)
+  * Swagger hide search field (#9554)
+  * Add "search" to reserved usernames (#9063)
+  * Switch to fomantic-ui (#9374)
+  * Only serve attachments when linked to issue/release and if accessible by user (#9340)
+  * Hide credentials when submitting migration through API (#9102)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```
(this is a backport)

File: CHANGELOG.md [hunk 2]
```diff
@@ -400,6 +413,7 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Update the provided gitea.service to mention socket activation (#8531)
   * Doc added how to setup email (#8520)
 * MISC
+  * Backport Locales [2020-01-14] (#9773)
```
Comment (techknowlogick) [inline CHANGELOG.md:415]:
```suggestion
```

**Review by techknowlogick — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jolheiser) [reply CHANGELOG.md:415]: This one I think we left because we don't usually backport locales. Is it significant enough to leave on the changelog?

Comment (techknowlogick) [reply CHANGELOG.md:415]: Ah yeah, I guess we also had this already in RC changelog. Ignore this nit. Thanks :)

**Review by 6543 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,13 +4,76 @@ This changelog goes through all the changes that have been made in each release
...
 
-## [1.11.0-RC2](https://github.com/go-gitea/gitea/releases/tag/v1.11.0-rc2) - 2020-01-18
+## [1.11.0](https://github.com/go-gitea/gitea/releases/tag/v1.11.0) - 2020-02-10
 * BREAKING
+  * Fix followers and following tabs in profile (#10202) (#10203)
   * Make CertFile and KeyFile relative to CustomPath (#9868) (#9874)
+  * Remove unused endpoints (#9538)
+  * Prefix all user-generated IDs in markup (#9477)
+  * Enforce Gitea environment for pushes (#8982)
+  * Hide some user information via API if user have no enough permission (#8655)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Hide some user information via API if user have not enough permissions (#8655)
```

**Review by guillep2k — APPROVED** [review]
The smallest nit

File: .changelog.yml [hunk 4]
```diff
@@ -1,4 +1,14 @@
+# The full repository name
```
Comment (jaqra) [inline .changelog.yml:1]: Not should be in another PR ❓ 🤔

Comment (6543) [reply .changelog.yml:1]: dont think this should be a problem - since this will get front-ported

Comment (jaqra) [reply .changelog.yml:1]: Just asking to understand behavior of working on gitea

Comment (6543) [reply .changelog.yml:1]: .changelog is config for the changelog-tool

Witch add a huge update - thats why the config was updated

**Review by lafriks — APPROVED** [review]
